From: Lennart Poettering <lennart@poettering.net>
Date: Tue, 11 Mar 2014 05:23:39 +0100
Subject: man: multiple sleep modes are to be separated by whitespace,
 not commas

As pointed out by Jason A. Donenfeld.

(cherry picked from commit 252094eb05c58270a0bc35b14ad30a126ddbb3bb)
---
 man/systemd-sleep.conf.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/man/systemd-sleep.conf.xml b/man/systemd-sleep.conf.xml
index a917f4d0..d0ea6d82 100644
--- a/man/systemd-sleep.conf.xml
+++ b/man/systemd-sleep.conf.xml
@@ -128,7 +128,7 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
         <citerefentry><refentrytitle>systemd-hibernate.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>, or
         <citerefentry><refentrytitle>systemd-hybrid-sleep.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>.
         More than one value can be specified by separating
-        multiple values with commas. They will be tried
+        multiple values with whitespace. They will be tried
         in turn, until one is written without error. If
         neither succeeds, the operation will be aborted.
         </para></listitem>
@@ -146,7 +146,7 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
         <citerefentry><refentrytitle>systemd-hibernate.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>, or
         <citerefentry><refentrytitle>systemd-hybrid-sleep.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>.
         More than one value can be specified by separating
-        multiple values with commas. They will be tried
+        multiple values with whitespace. They will be tried
         in turn, until one is written without error. If
         neither succeeds, the operation will be aborted.
         </para></listitem>
